cljfmt/src/cljfmt/stacktrace.clj

```diff
@@ -0,0 +1,61 @@
+;;;   Copyright (c) Rich Hickey. All rights reserved.
```

Comment:
Let's change the namespace name to something like `cljfmt.impl.stacktrace` to indicate it's not for public use.